### ISSUE TITLE
Launch an xdg-open custom URI

### ISSUE THREAD

**Issue by canadaduane** [pop-os/launcher#154]
The https://obsidian.md app (flatpak) registers its own mime type, `obsidian://...`. From the CLI, I can launch Obsidian with an open search page, like so:

```
$ xdg-open obsidian://search?query=journal
```

I can add a launcher config entry, like so:

```
$ sudo vim /usr/lib/pop-launcher/plugins/web/config.ron

#
#    (
#       matches: ["j", "journal"],
#       queries: [(name: "Journal", query: "obsidian://search?query=")]
#    ),
#
```

However, the launcher seems to open a browser to `https://obsidian://search?query=`.

Is this intended behavior?

**Comment by mmstick** [comment]
It's not. Would you like to also patch this?

**Comment by mmstick** [comment]
I guess it's kind of weird for the web plugin to be used for non-web URIs.

**Comment by canadaduane** [comment]
Is there another plugin that would be better suited to this kind of URI?

**Comment by canadaduane** [comment]
Since the `web` plugin does use xdg-open already, it does seem very adjacent. I will look into patching unless you feel it does not belong here.

**Comment by mmstick** [comment]
There's not another plugin

**Comment by canadaduane** [comment]
The above PR gets a step closer to my goal; however, I've now discovered a new impediment:

```
Jan 10 20:45:32 rosie gnome-shell[15076]: got query: obsidian://search?query=test
Jan 10 20:45:32 rosie systemd[3774]: Started app-flatpak-md.obsidian.Obsidian-15102.scope.
Jan 10 20:45:32 rosie gnome-shell[15111]: /app/bin/obsidian.sh: line 27: echo: write error: Broken pipe
```

I'm still investigating, but it appears something is not long-lived enough for this to work as-is.

**Comment by canadaduane** [comment]
> I'm still investigating, but it appears something is not long-lived enough for this to work as-is.

Fixed in #155

**Comment by jacobgkau** [comment]
If the PR fixing the issue hasn't been merged yet, then please don't close it.
